src/Client/Web/Bit.Client.Web.BlazorUI/Components/DropDown/BitDropDown.Fluent.scss

```diff
@@ -289,6 +289,41 @@
     }
 }
 
+.overflow-hidden {
+    overflow: hidden !important;
+}
+
+@media only screen and (min-width: 0) and (max-width: rem(639px)) {
+    .bit-drp-items-wrapper {
+        box-shadow: rgb(0 0 0 / 22%) 0 25.6px 57.6px 0, rgb(0 0 0 / 18%) 0 4.8px 14.4px 0;
+        position: fixed !important;
+        top: 0 !important;
+        left: unset !important;
+        right: 0 !important;
+        bottom: unset !important;
+        height: 100% !important;
+        width: rem(272px) !important;
```

Comment:
plz remove these `!important`s :)))

Reply:
Here if I want to remove the `!important` from the styles I should handle it with TS/JS but its not as efficient as css media queries,
I give you an example, consider a mobile user opens a dropdown and then rotate the phone to be in the landscape mode in this case the media query will perform correctly but in TS/JS approach the user must close and re-open the dropdown to see it in proper way

Reply:
in these situations, we can simply close the callout. we just need to capture the appropriate events.

Reply:
we did the same in the window resize event.

Reply:
But in Fluent's Demos the dropdown does not close on rotate